### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sonpxp**
has commit

### CODE REVIEW

**Review by dookki — CHANGES_REQUESTED** [review]
change desc

**Review by sonpxp — COMMENTED** [review]
req

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # Google-Translate
 Tool translate language, Translate strings.xml in android studio
```
Comment (sonpxp) [inline README.md:2]: miss

**Review by sonpxp — COMMENTED** [review]
oke

**Pull request closed without merging**